deformable_manipulation_experiment_params/include/deformable_manipulation_experiment_params/ros_params.hpp

```diff
@@ -168,8 +179,9 @@ namespace smmap
         {
             case DeformableType::ROPE:
                 // TODO: why did Dmitry's code use 0.5f here?
-                return ROSHelpers::GetParamDebugLog(nh, "rope_gripper_apperture", 0.03f);
-
+                // Mengyao: was 0.03f
+            //    return ROSHelpers::GetParamDebugLog(nh, "rope_gripper_apperture", 0.029f);
+            return ROSHelpers::GetParam(nh, "rope_gripper_apperture", 0.03f);
```

Comment:
Is there any reason this was changed? Any reason not to remove the extra comments and just leave one version of the code here (with the right tabbing).

Reply:
I changed it when I was working on the squeezing boxes for the grippers. The comment line can be removed.